### PR TITLE
[terra-responsive-element] Doc Update

### PR DESCRIPTION
**Pull request by MadanKumarGovindaswamy**
### Summary

**What was changed:**
Updated hyperlinks to terra-breakpoints

**Why it was changed:**
Hyperlink was not working

### Testing

This change was tested using:

- [ ] WDIO
- [ ] Jest
- [ ] Visual testing (please attach a screenshot or recording)
- [ ] Other (please describe below)
- [x] No tests are needed

### Reviews

In addition to engineering reviews, this PR needs:

- [ ] UX review
- [ ] Accessibility review
- [x] Functional review

### Additional Details

**This PR resolves:**

UXPLATFORM-7500 

---

Thank you for contributing to Terra.
@cerner/terra

### CODE REVIEW

File: packages/terra-responsive-element/src/ResponsiveElement.jsx [hunk 1]
```diff
@@ -22,27 +22,33 @@ const propTypes = {
    */
   onResize: PropTypes.func,
   /**
-   * [Deprecated] An element to be displayed at tiny breakpoints.
```
Comment (sdadn) [inline packages/terra-responsive-element/src/ResponsiveElement.jsx:25]: Why is the deprecation badge removed?

Comment (sdadn) [reply packages/terra-responsive-element/src/ResponsiveElement.jsx:25]: Discard this, I didn't see the full url of the update 🙂.

File: packages/terra-responsive-element/CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+* Added
+  * Added deprecated icon in props table
```
Comment (sdadn) [inline packages/terra-responsive-element/CHANGELOG.md:6]:
```suggestion
  * Added deprecated icon in props documentation.
```

File: packages/terra-core-docs/CHANGELOG.md [hunk 3]
```diff
@@ -14,6 +14,7 @@
   * Updated `terra-status` documentation with deprecation notice and badge.
   * Updated `terra-html-table` documentation with deprecation notice and badge.
   * Updated test and doc examples for `terra-progress-bar`
+  * Updated `terra-responsive-element` document to have correct hyperlink to `terra-breakpoints`
```
Comment (saket2403) [inline packages/terra-core-docs/CHANGELOG.md:17]: Move this to unreleased.

**Review by saket2403 — APPROVED** [review]

**Pull request merged**